packages/perseus/src/widgets/interactive-graphs/graphs/segment.tsx

```diff
@@ -54,46 +59,52 @@ const SegmentGraph = ({dispatch, graphState}: SegmentProps) => {
     }
 
     return (
-        <g
-            aria-label={wholeSegmentAriaLabel}
-            aria-describedby="segment-description"
-        >
+        <>
```

Comment:
I think we might still want a label on the overall segments graph saying something like "[Num] segments" so the user knows what to expect before going through each segment individually.